scripts/protobuf.go

```diff
@@ -115,12 +116,26 @@ func install(deps ...string) error {
 func generate(dir string, dirs []string, files []string) error {
 	defer switchdir(dir)()
 
+	cmd := exec.Command("protolock", "status")
+	local, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	cmd.Dir = findProtolockDir(local)
+	out, err := cmd.CombinedOutput()
+	if len(out) > 0 {
+		fmt.Println(string(out))
+	}
+	if err != nil {
```

Comment:
maybe doesn't matter but why is the error check happening after the len(out) check?

Reply:
Because we want to output the content why it failed, usually it contains useful error message what the actual error is.